Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:6
+
+WORKDIR /opt
+
+ENV JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk.x86_64
+
+RUN yum update -y && yum install -y wget \
+  && wget http://ftp.rediris.es/mirror/fedora-epel/6/i386/epel-release-6-8.noarch.rpm && yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm \
+  && yum install -y npm git unzip java-1.7.0-openjdk java-1.7.0-openjdk-devel tomcat \
+  && wget -c http://ftp.cixug.es/apache/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.zip && unzip -oq apache-maven-3.2.5-bin.zip \
```

Comment:
You should chose a more "neutral" mirror. Not everybody downloading/using this is located in Spain ;)